bundles/org.palladiosimulator.analyzer.slingshot.core/src/org/palladiosimulator/analyzer/slingshot/core/engine/SimulationEngineSSJ.java

```diff
@@ -74,6 +74,7 @@ public void stop() {
 		simulator.stop();
 		this.eventBus.acceptEvents(false);
 		this.isAcceptingEvents = false;
+		eventBus = Bus.instance(); // reset bus
```

Comment:
I feel like that the event bus should have some kind of "re-initialization" mechanism. In fact, it is possible to unregister objects/listeners from it already, but maybe a single method that unregisters all known objects. Though I think that this should be ok for now.

Reply:
i left it at that kinda hacky solution, 'cuz i also considered, whether it makes sense to create an entirely new engine (and driver) for each simulation run. but that's something better discussed in person.  

for now, an unregisterAll would be nice though.